src/v/config/configuration.cc

```diff
@@ -470,6 +471,13 @@ configuration::configuration()
       "write with the given producer id.",
       {.visibility = visibility::user},
       10080min)
+  , max_saved_pids_count(
+      *this,
+      "max_saved_pids_count",
```

Comment:
this is very generic name, can we make it more specific ?

Reply:
I am not sure about naming?
Smth like `max_rm_stm_state_size`?